### ISSUE TITLE
Home: Missing Text Within Search Submit Button

### ISSUE THREAD

**Issue by jscholes** [benetech/ImageShare#57]
The button to submit a search does not have a visible textual label (such as "Search") appearing next to the magnifying glass icon, to make the purpose completely clear to all users and ensure that it can be easily activated by speech recognition users.

![Screenshot of the magnifying glass search submit icon button.](https://user-images.githubusercontent.com/6454755/80002303-12412f00-8485-11ea-8881-e818597c2b40.png)

**Comment by clapierre** [comment]
Absolutely correct, we can add "Search" beside that Icon for Dictation and Cognitive concerns.

Question, since "Search" will now appear directly after this image, would the image be set to role="presentation" alt="" so a SR user won't hear search twice, or would we keep the alt text "Search" on the image/button so that it is announced together and make the Text beside it be only visual and hidden to AT to deal with speaking "Search" twice?

**Comment by jscholes** [comment]
@clapierre Empty `alt` on the image will do it.  Or `aria-hidden="true"` if implemented as an SVG.  Then the inner text is the only thing contributing to the accessible name.  Guarantees that they stay in sync.

**Comment by clapierre** [comment]
So the accessibility name will be derived from the text that is visible right?  So that if we ever do want to  translated into another language that won't be a problem.

**Comment by sinabahram** [comment]
This is handled in new designs and also implementation .
